static/app/views/settings/components/teamSelect.tsx

```diff
@@ -92,6 +92,41 @@ function TeamSelect({
   const {teams, onSearch, fetching} = useTeams();
   const {orgRoleList, teamRoleList} = organization;
 
+  const slugsToFilter =
+    selectedTeams?.map(tm => tm.slug) || selectedTeamRoles?.map(tm => tm.teamSlug) || [];
+
+  // determine if adding a team changes the minimum team role
+  const getTopOrgRole = () => {
+    const teamsWithOrgRole = teams.filter(
+      team => slugsToFilter.some(slug => slug === team.slug) && team.orgRole !== null
+    );
+    // get teams with org roles, if any
+    const teamOrgRoles =
```

Comment:
can we name this const `teamsWithOrgRoles` then?

Reply:
Oh lol. you already named the previous const that 🤔 

can we name the previous array `filteredTeams` instead?